src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/text-unit-form/text-unit-form.component.ts

```diff
@@ -21,30 +22,36 @@ export interface TextUnitFormData {
     styles: [],
 })
 export class TextUnitFormComponent implements OnInit, OnChanges, OnDestroy {
-    @Input()
-    formData: TextUnitFormData;
+    protected readonly faTimes = faTimes;
+
+    @Input() formData: TextUnitFormData;
 
     @Input() isEditMode = false;
     @Output() formSubmitted: EventEmitter<TextUnitFormData> = new EventEmitter<TextUnitFormData>();
 
-    @Input()
-    hasCancelButton: boolean;
-    @Output()
-    onCancel: EventEmitter<any> = new EventEmitter<any>();
-
-    faTimes = faTimes;
+    @Input() hasCancelButton: boolean;
+    @Output() onCancel: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
Same as above, re-ordered Inputs and Ouputs and constants

Inputs and Outputs are written in one line, as we usually do it throughout the application